scripts/write_deployinst_input.py

```diff
@@ -319,6 +356,8 @@ def region_render(list, template, full_template, output_file):
 
     """
 
+    template = read_template('../templates/deployinst_template.xml.in')
+    full_template = read_template('../templates/region_output_template.xml.in')
```

Comment:
why did this stop being a parameter? It was much better practice for this to be a function parameter!

Reply:
My thought is that it was much better practice for this to be a function parameter.